less/mixins.less

```diff
@@ -1 +1,8 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.btn-style(@padding, @border-radius, @background-color, @color, @cursor) {
```

Comment:
mixin looks great! don't forget that you can set default values to your arguments so you don't need to pass them every time, for example you could write @padding: 1rem, @border-radius: .5rem on line 2 so when you call .flex you wouldn't need to pass values for those properties. You're also still able to pass in new values like you would if you hadn't set defaults.